src/reference/asciidoc/dsl.adoc

```diff
@@ -115,28 +115,37 @@ The endpoints are automatically wired together by using direct channels.
 
 [[java-dsl-class-cast]]
 .Lambdas And `Message<?>` Arguments
-IMPORTANT: When using lambdas in EIP methods, the "input" argument is generally the message payload.
+[IMPORTANT]
+====
+When using lambdas in EIP methods, the "input" argument is generally the message payload.
 If you wish to access the entire message, use one of the overloaded methods that take a `Class<?>` as the first parameter.
 For example, this won't work:
 
-====
 [source, java]
 ----
 .<Message<?>, Foo>transform(m -> newFooFromMessage(m))
 ----
-====
 
 This will fail at runtime with a `ClassCastException` because the lambda doesn't retain the argument type and the framework will attempt to cast the payload to a `Message<?>`.
 
 Instead, use:
 
-====
 [source, java]
 ----
 .(Message.class, m -> newFooFromMessage(m))
 ----
 ====
```

Comment:
Unbalanced fence removal?

Why remove the fences at all?

Reply:
I removed it because they don't work in the `[IMPORTANT]` block. This piece of code has to be as a part of the one.
Right now it looks like:
![default](https://user-images.githubusercontent.com/1071590/50170378-cd6c1d80-02bd-11e9-96a5-0eff53dfcbf8.png)

Not perfect, but still...

Reply:
Oh; got it; didn't notice the new fence at the top.